### PR TITLE
Styling tombstone

### PR DESCRIPTION
**Pull request by Braun-Johannes**

### CODE REVIEW

**Review by ChristianMoschke — CHANGES_REQUESTED** [review]
I've run into some things.

1. **Time Alive Reset Issue:** After killing a pet and reloading the living room while the tombstone is displayed, the `time alive` gets reset to `0s`.

2. **File Naming Suggestion:** Consider renaming `TombstoneButton.js` to `Tombstone.js` to reduce confusion.

3. **Styling Consistency:** In `TombstoneButton.js`, review all `styling components` for values like `0px` and change them to `0`. While using `0px` may not break the code, it is not a common practice.

File: components/TombstoneButton.jsx [hunk 1]
```diff
@@ -1,25 +1,94 @@
+import styled from "styled-components";
 import PositionedButton from "@/components/Styles/StyledButton";
-import StyledDiv from "@/components/Styles/StyledDiv";
+import SVGIcon from "./SVGIcon";
```
Comment (SabinePfanner) [inline components/TombstoneButton.jsx]: No blocker, just a hint for the future - try to keep your import style consistent e.g. either with ```@``` or ```.```

File: components/TombstoneButton.jsx [hunk 2]
```diff
@@ -1,25 +1,94 @@
...
-            Click here <br /> to select a
-            <br /> new Survival Pet
-          </p>
-        </PositionedButton>
-      </StyledDiv>
+          <br />
+          <PositionedButton
+            onClick={onDeletePet}
+            $variant="tombstone"
+            position="relative"
```
Comment (SabinePfanner) [inline components/TombstoneButton.jsx:34]: Just a question : I guess "position" is just a prop of the Styled Element ```<PositionedButton>```  named similar to the CSS styling element?

File: components/TombstoneButton.jsx [hunk 3]
```diff
@@ -1,25 +1,94 @@
...
+          </PositionedButton>
+        </LowerTombstone>
+      </StyledTombstone>
     </>
   );
 }
+const StyledTombstone = styled.div`
+  display: inline-block;
+  text-align: center;
+  font-size: 0;
```
Comment (SabinePfanner) [inline components/TombstoneButton.jsx:57]: ```Font-size: 0```implements, that there is no visible text but the component does not include any text. As you even style the text in the styled component ```LowerTombstone``` already separately, I guess this styling can be removed here.

File: components/TombstoneButton.jsx [hunk 4]
```diff
@@ -1,25 +1,94 @@
...
+
+const UpperTombstone = styled.div`
+  border-radius: 50%;
+  width: 200px;
+  height: 90px;
+  background: grey;
+  margin: 0 auto;
+  position: relative;
+  top: 65px;
+  z-index: -10;
```
Comment (SabinePfanner) [inline components/TombstoneButton.jsx:70]: Isn't there any other solution than using a negative ```z-index```? ;)

**Review by SabinePfanner — APPROVED** [review]
LGMT, only some small hints but no blockers

File: components/TombstoneButton.jsx (hunk 2 above)
Comment (niehaus-alexander) [reply components/TombstoneButton.jsx:34]: yep indeed

**Pull request closed without merging**